src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        key={movie.imdbId}
+        title={movie.title}
+        description={movie.description}
+        imgUrl={movie.imgUrl}
+        imdbUrl={movie.imdbUrl}
+        imdbId={movie.imdbId}
+      />
```

Comment:
you can just pass the movie and destruct it in the `MovieCard`. It will look simpler, then you shouldn't map all types from `Movie`